stacks-in-depth/nodes-and-miners/mine-testnet-stacks-tokens.md

```diff
@@ -94,26 +92,58 @@ After this runs, you should see some JSON printed to the screen that looks like
 
 ```json
 {
-  "mnemonic": "exhaust spin topic distance hole december impulse gate century absent breeze ostrich armed clerk oak peace want scrap auction sniff cradle siren blur blur",
+  "mnemonic": "put cushion unique anxiety tooth brand record call toy able artwork actual disagree round drum equip surprise injury matter merge desk flash service rather",
   "keyInfo": {
-    "privateKey": "2033269b55026ff2eddaf06d2e56938f7fd8e9d697af8fe0f857bb5962894d5801",
-    "address": "STTX57EGWW058FZ6WG3WS2YRBQ8HDFGBKEFBNXTF",
-    "btcAddress": "mkRYR7KkPB1wjxNjVz3HByqAvVz8c4B6ND",
+    "privateKey": "3d17ffc45de1682a076a294753335cae5531d8ea50d3ca67a84e00cb6c84cf4d01",
+    "publicKey": "0299445e0a04d9d7b9dcf85837960c6b7da279d1dc2c1f632de2e5785b56feb885",
+    "address": "ST1P67504RRJBG85ZG7MVZX6WVEG5Q766YD97R65H",
+    "btcAddress": "mqQ4ZaxBu5CP5K8yuvMnS3PoELb52N1h8Q",
+    "wif": "cPdTdMgww2njhnekUZmHmFNKsWAjVdCR4cfvD2Y4UQhFzMmwoW33",
```

Comment:
same as above - use the same values as miner-docs repo, or create new fake ones.